demo/package.json

```diff
@@ -9,7 +9,9 @@
     "serve": "astro preview"
   },
   "devDependencies": {
-	"@astropub/flow": "0.3.6",
     "astro": "latest"
+  },
+  "dependencies": {
+    "@astropub/flow": "workspace:^0.3.6"
```

Comment:
This is used to test the local package in the repo

Reply:
Yes. I can see the reason you would want PNPM for this.

I want to see if there’s a way I can have both. Would you know if we can have it both ways, so that NPM install and PNPM can both work as expected by their own capabilities?

I’ll ask around this morning (my timezone is -5 ET).

Reply:
I don't know. I'm not an expert on this stuff. I think npm can be used on node_modules that were created by pnpm.